web/src/message_edit.ts

```diff
@@ -1413,8 +1420,11 @@ export function maybe_show_edit($row: JQuery, id: number): void {
     }
 
     if (currently_editing_messages.has(id)) {
-        const $message_edit_content = currently_editing_messages.get(id);
-        edit_message($row, $message_edit_content?.val() ?? "");
+        const message_edit_content = currently_editing_messages.get(id)?.val();
```

Comment:
Is this refactor a functional change, or just trying to simplify the variables? Might be better done as a prep commit.

I guess the next line is the functional change, replacing `edit_message` with `start_edit_with_content`?

Reply:
I can't quickly answer that -- use your best judgement and post and explanation of your reasoning (or brainstorm with your mentor).